pkg/util/ballast.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+// AllocateBallast allocates ballast of given size. to alter GC behaviour. See https://github.com/golang/go/issues/23044
```

Comment:
[Nit] I guess there should be a comma.
```suggestion
// AllocateBallast allocates ballast of given size, to alter GC behaviour. See https://github.com/golang/go/issues/23044
```